fecgov/fec-pattern-library
# Remove duplicate values

## Summary of changes

- Addresses #198 
_Reverts back to before the npm upgrade, adds back all recent changes (circleci docker images, readme edits) then upgrades moment with npm 6.14.15. 


This was complicated by the fact that our pattern library pulls in all CMS dependencies and we currently can't rebuild our node_modules and package-lock file without NPM errors. We are out of sync with CMS and will eventually need to be aligned back with CMS. 
 

## How to test
-`gh pr checkout 202`
-`nvm install 14.17` ( should be npm version 6.14.15  if not `npm install -g npm@6.14.15`)
-`npm i && npm run build`
-`npm run start`
